### ISSUE TITLE
add visibility toggle for overlay in completeness mission type

### ISSUE THREAD

**Issue by danbjoseph** [mapswipe/mapswipe#673]
### Background 
Céline reached out via the MapSwipe Slack: 
> Hello everyone, I'm chatting with a semi autonomous institution in Mexico, the Comisión Nacional para el Conocimiento y Uso de la Biodiversidad o National Commission for the Knowledge and Use of Biodiversity (CONABIO), an institution visionary in term of introduction of citizen science in the environmental monitoring, and leader in that domain in the region. Around the 2 little pilots we have created at HOT in Mapswipe to monitor forest loss in Mexico and Central America, I looked for their specialists' guidance and they are interested to monitor specific ecosystems at risk: the mangrove line on all the coastline that receives the hurricanes, and then Central America. They are interested in comparing mangrove changes during time. They don't have a different access than our to satellite images but propose to compare a thematic map of a past date with a recent image, helping to see where is the mangrove limit and receive data of changes in these limits. They also propose to give a workshop for that. What do you think, is it technically pertinent and possible?   They are also interested on tracking burnt areas in forest.

After conversations with CONABIO, we reviewed all 3 existing mission types in relation to their use-case. They have a layer showing the historical extent of mangroves, and they want users to help identify where mangroves have been lost or destroyed. For example, filled in for building construction or cleared for beaches. 

The completeness mission seemed to be the best fit. With 2 main issues:
- **Issue with color** - The squares turn green/yellow/red for 1/2/3 taps and none of those colors overlay the red mangrove layer well. Benni says “we've tested completeness in the past with buildings in blue and higher transparency.” But in those cases the layer would never cover the whole screen like in this use-case. It would be lots of individual smaller buildings. That is, the entire screen would never be filled by a single building, but it may be entirely filled by a landcover extent. With a transparent blue overlay covering the whole screen, the user might think it is just the color of the satellite imagery.
- **Issue with transparency** - Right now, the red mangrove layer is very hard to see through. Increasing the transparency might help, but could also make it harder to see the extent. 
<img width="763" alt="Screen Shot 2023-01-16 at 9 17 25 AM" src="https://user-images.githubusercontent.com/4806884/212699131-58a463f7-1fa0-47ee-be02-6dda84e487f7.png">

### Proposal 
Proposing to add a visibility toggle for the overlay in completeness mission. I think this would be beneficial for the landcover use-case described here, as well as for when using this for buildings or some such feature collection.
- This should be possible by replicating the existing system from the footprint validation project type.
- The toggle can go in the bar below the image opposite the scale bar.
- For the footprint validation, discussion occurred around  the position of the toggle between left/right/center with respect to how users hold their phone (left/right hand). Left favors right-handers. Right favors left-handers.  What about immediately to the right of the scale bar? It would be mostly centered with a a slight favor to right-handers?

I think #672 (use icon watermarks for the number of taps) would also benefit this mission type, in order to mitigate the effects of 2 transparent layers interacting.

**Comment by Monwaruli** [comment]
I would like to work on this.

**Comment by danbjoseph** [comment]
hi @Monwaruli, thanks for your interest. a good first step would be to write up a rough explanation of what changes you plan to do to accomplish this. perhaps a wireframe visualizing the changes. then you can get the approval of the project team before moving ahead. or, if you think it's not that much time then you can go ahead with the changes and submit a pull request. 
have you taken a look at the [contributing guide](https://github.com/mapswipe/mapswipe/blob/master/CONTRIBUTING.md)?

**Comment by Monwaruli** [comment]
> hi @Monwaruli, thanks for your interest. a good first step would be to write up a rough explanation of what changes you plan to do to accomplish this. perhaps a wireframe visualizing the changes. then you can get the approval of the project team before moving ahead. or, if you think it's not that much time then you can go ahead with the changes and submit a pull request. have you taken a look at the [contributing guide](https://github.com/mapswipe/mapswipe/blob/master/CONTRIBUTING.md)?

I sent an email yesterday to info@mapswipe.org about contributing for an assignment for my class. I would greatly appreciate it if you can take a look at that.

**Comment by Monwaruli** [comment]
> @Monwaruli I've emailed you privately with the file you need. Please confirm if everything is ok with that.

Yes, I received the email, and joined the Slack three weeks ago. I have been working towards solutions to some of the issues for the app.
